ivy_tests/test_ivy/test_frontends/test_numpy/test_manipulation_routines/test_changing_kind_of_array.py

```diff
@@ -32,3 +32,32 @@ def test_numpy_asarray(
         a=x,
         dtype=input_dtype[0],
     )
+
+
+@handle_frontend_test(
```

Comment:
Please replace this decorator with `@handle_frontend_function` following a recent refactoring.